docs/src/styles/docs/tokenList.scss

```diff
@@ -0,0 +1,151 @@
+@import './_variables.scss';
+
+.docs-tokenGroup {
+  display: grid;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+  --swatch-size: 64px;
+  @media (min-width: $breakpoint-medium) {
+    --swatch-size: 32px;
+  }
+}
+
+.docs-tokenGroup--list {
+  li {
+    border-bottom: 1px solid var(--amplify-colors-neutral-20);
+    padding: var(--amplify-space-small) 0;
+    &:last-child {
+      border-bottom: none;
+    }
+  }
+}
+
+.docs-tokenGroup--grid {
+  grid-template-columns: repeat(auto-fill, minmax(16rem, 1fr));
+  gap: var(--amplify-space-large) var(--amplify-space-medium);
+}
+
+.docs-tokenItem {
+  display: grid;
+  line-height: var(--amplify-line-heights-small);
+  gap: var(--amplify-space-xxxs) var(--amplify-space-medium);
+  align-items: center;
+}
+
+.docs-tokenItem--colors {
+  grid-template-columns: auto 1fr;
+  .docs-colorBlock {
+    grid-area: 1 / 1 / 4 / 1;
+    align-self: center;
+  }
+  @media (min-width: $breakpoint-medium) {
+    grid-template-columns: var(--swatch-size) 180px 1fr 1fr;
+    padding: var(--amplify-space-xs) 0;
+    align-items: center;
+    .docs-colorBlock {
+      grid-area: 1 / 1 / 2 / 1;
+    }
+  }
+}
+
+.docs-tokenItem--fontWeights, .docs-tokenItem--fontSizes, .docs-tokenItem--lineHeights {
+  grid-template-columns: auto 1fr;
+  .docs-fontBlock {
+    grid-area: 1 / 1 / 4 / 1;
+    align-self: center;
+  }
+  @media (min-width: $breakpoint-medium) {
+    grid-template-columns: auto 140px 180px 1fr;
+    .docs-fontBlock {
+      grid-area: 1 / 1 / 2 / 1;
+    }
+  }
+}
+
+.docs-tokenItem--space, .docs-tokenItem--borderWidths {
+  grid-template-columns: 100px auto;
+  @media (min-width: $breakpoint-medium) {
+    grid-template-columns: 140px 140px 80px auto;
+    .docs-tokenItem-path {
+      text-align: right;
+    }
+  }
+}
+
+
+.docs-tokenItem--radii {
+  grid-template-columns: auto;
+  justify-items: center;
+  .docs-radiusBlock {
+    grid-area: 1 / 1 / 4 / 1;
+    align-self: center;
+  }
+  @media (min-width: $breakpoint-medium) {
+    // This first column size accomodates our largest radius + an extra
+    // pixel width to account for SVG stroke
+    grid-template-columns: calc(2 * var(--amplify-radii-xxxl) + 1px) 100px 100px 1fr;
+    justify-items: start;
+    .docs-radiusBlock {
+      grid-area: 1 / 1 / 2 / 1;
+      margin-left: auto;
+    }
+  }
+}
+
+.docs-tokenItem--fontFamily {
+  .docs-fontBlock {
+    padding: var(--amplify-space-medium) 0;
+  }
+}
+
+
+.docs-colorBlock {
+  width: var(--swatch-size);
+  height: var(--swatch-size);
+  border: 1px solid rgba(0,0,0,.1);
```

Comment:
Can we use `var(--amplify-colors-overlay-xx)` for border color? Or maybe adjust it to 20. That way the border color can be flipped in dark mode as well. It is not friendly to some color in dark mode now.

![Screen Shot 2022-06-09 at 3 18 16 PM](https://user-images.githubusercontent.com/40295569/172954704-507d67b1-a39a-45cb-bdb7-3f32270c8b05.png)


With `var(--amplify-colors-overlay-20)`

![Screen Shot 2022-06-09 at 3 17 52 PM](https://user-images.githubusercontent.com/40295569/172954664-6761d1b1-808b-469e-86e2-20f03b10bba7.png)